### PR TITLE
change: use new pane-based paradigm layouts for all paradigms

### PR DESCRIPTION
**Pull request by eddieantonio**
# What's in this PR?

 - **added** "basic" sized paradigms for all noun and verb word classes
 - **changed**: all paradigms use the new pane-based paradigm subsystem!
 - **changed** renamed "NAD" → "NDA", "NID" → "NDI"
 - **changed** Cypress tests to make fewer assumptions!

@aarppe, please have a look at the basic paradigm layouts.

### CODE REVIEW

File: cypress/integration/regressions.spec.js [hunk 1]
```diff
@@ -155,10 +155,10 @@ context('Regressions', () => {
       .click()
 
     // N.B.: This test will fail if the word changes. ¯\_(ツ)_/¯
-    cy.contains('th', 's/he → him/her/them (further)') // 3Sg -> 4 (lemma)
-    cy.contains('th', 's/he → me')                  // 3Sg -> 1Sg
-    cy.contains('th', 's/he → you (one)')           // 3Sg -> 2Sg
-    cy.contains('th', 's/he/they (further) → him/her') // 4 -> 3
+    cy.contains('th', 's/he → him/her/them') // 3Sg -> 3' (lemma)
```
Comment (aarppe) [inline cypress/integration/regressions.spec.js:158]: For consistency's sake, we should use either `3Sg -> 4Sg/PlO` (or `3s -> 4` or `3s -> 3'`).

File: cypress/integration/regressions.spec.js [hunk 2]
```diff
@@ -155,10 +155,10 @@ context('Regressions', () => {
...
+    cy.contains('th', 's/he → me')           // 3Sg -> 1Sg
+    cy.contains('th', 's/he → you')          // 3Sg -> 2Sg
+    cy.contains('th', 's/he/they → him/her') // 3' -> 3
```
Comment (aarppe) [inline cypress/integration/regressions.spec.js:161]: For consistency's sake, we should use either `4Sg/PlO -> 3Sg` (or `4 -> 3s` or `3' -> 3s`).

**Review by aarppe — COMMENTED** [review]
Looked through the layouts, and looks good; made two comments concerning notation in code comments.

File: cypress/integration/paradigm.spec.js [hunk 3]
```diff
@@ -91,11 +91,14 @@ describe('I want to know if a form is observed inside a paradigm table', () => {
   })
 })
 
-describe(' I want to see a clear indicator that a form does not exist', () => {
+describe('I want to see a clear indicator that a form does not exist', () => {
+  it('shows cells that does not exist as a em dash', () => {
```
Comment (andrewdotn) [inline cypress/integration/paradigm.spec.js:95]:
```suggestion
  it('shows cells that do not exist as an em dash', () => {
```

File: src/CreeDictionary/CreeDictionary/views.py [hunk 4]
```diff
@@ -381,13 +381,50 @@ def paradigm_for(
...
+def convert_crkeng_paradigm_size_to_size(paradigm_size: ParadigmSize):
+    """
+    Returns the crkeng layout size, which is currently:
+     - basic
+     - full
+    """
+    return {
+        ParadigmSize.FULL: "full",
+        ParadigmSize.BASIC: "basic",
+        # The linguistic "size" does not, as of yet exist, however its contents is
```
Comment (andrewdotn) [inline src/CreeDictionary/CreeDictionary/views.py:409]:
```suggestion
        # The linguistic "size" does not, as of yet exist, however its content is
```

**Review by andrewdotn — APPROVED** [review]
Looks good!

**Comment by aarppe** [comment]
This is an excellent development!

**Pull request merged**